front/migrations/20240731_backfill_views_in_agent_data_source_configurations.ts

```diff
@@ -20,7 +21,7 @@ async function backfillViewsInAgentDataSourceConfigurationForWorkspace(
   const dataSources = await DataSourceResource.listByWorkspace(auth);
 
   // Filter managed data sources.
-  const managedDataSources = dataSources.filter((ds) => ds.isManaged());
+  const managedDataSources = dataSources.filter((ds) => isManaged(ds.toJSON()));
```

Comment:
Any chance we can keep `isManaged` on the data source and call the underlying implementation of the typeguard?

Reply:
done !